CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### Version 9.3
 * Adds `FallbackFactory`, allowing access to the cause of a Hystrix fallback
+* Adds support for encoded parameters in @Param via @Param(encoded = true)
```

Comment:
simplify and format? Ex.

* Adds support for encoded parameters via `@Param(encoded = true)`